app/models/recommendation.rb

```diff
@@ -24,4 +24,8 @@ class Recommendation < VersionedRecord
 
   validates :title, presence: true
   validates :reference, presence: true, uniqueness: true
+
+  def is_current
+    categories.any?(&:is_current)
+  end
```

Comment:
this is too strict: a rec is also considered "current" when it does not have a link with a "reporting cycle" category 

see https://github.com/impactoss/impactoss-tasks/issues/13#issuecomment-2157599821

Reply:
likewise, we could limit the categories we check for "is_current" to reporting cycles (taxonomy id = current_reporting_cycle)

Reply:
> likewise, we could limit the categories we check for "is_current" to reporting cycles (taxonomy id = current_reporting_cycle)

that's already handled inside category (it's the first thing that's checked) so I believe recommendation doesn't have to know that part

Reply:
> this is too strict: a rec is also considered "current" when it does not have a link with a "reporting cycle" category

Fixed in 1c07069e07cf3ff95bfc01c7b49353ab6e658c5c